doc/tutorials/basic-minting-policies.rst

```diff
@@ -19,15 +19,18 @@ The minting policy is a function which receives these two inputs as *arguments*.
 The validating node is responsible for passing them in and running the minting policy.
 As with validator scripts, the arguments are passed encoded as :hsobj:`PlutusCore.Data.Data`.
 
-Using the script context
-------------------------
+Plutus V1 and V2 script context
```

Comment:
You could say "Plutus script context versions" just so it remains current when we have V3 etc.